### PR TITLE
[16.0][UPD] stock_request: README.rst typo

### PR DESCRIPTION
**Pull request by abbasEbadian**

### CODE REVIEW

File: stock_request/README.rst [hunk 1]
```diff
@@ -80,7 +80,7 @@ also cancelled.
 Known issues / Roadmap
 ======================
 
-There is no way to achieve Storck Request and Stock Request Orders.
+There is no way to archive Stock Request and Stock Request Orders.
```
Comment (rousseldenis) [inline stock_request/README.rst]: @abbasEbadian Please change this in readme fragments instead.

File: stock_request/README.rst [hunk 2]
```diff
@@ -138,4 +138,4 @@ promote its widespread use.
 
 This module is part of the `OCA/stock-logistics-request <https://github.com/OCA/stock-logistics-request/tree/16.0/stock_request>`_ project on GitHub.
 
-You are welcome to contribute. To learn how please visit https://odoo-community.org/page/Contribute.
+You are welcome to contribute. To learn how please visit https://odoo-community.org/page/contribute.
```
Comment (rousseldenis) [inline stock_request/README.rst]: This will be regenerated automatically. That said, the link is correct.

**Review by rousseldenis — CHANGES_REQUESTED** [review]

File: stock_request/README.rst (hunk 1 above)
Comment (abbasEbadian) [reply stock_request/README.rst]: Done.

**Comment by LoisRForgeFlow** [comment]
@abbasEbadian Could you squash all your commits into a single one?

**Pull request closed without merging**